src/topology/algebra/group.lean

```diff
@@ -326,10 +326,10 @@ topological_space.nhds_mk_of_nhds _ _
     begin
       refine ⟨(λx:α, x + b) '' t, image_mem_map ht, _, _⟩,
       { refine set.image_subset_iff.2 (assume b hbt, _),
-        simpa using eqt 0 b t0 hbt },
+        simpa using eqt 0 t0 b hbt },
```

Comment:
It looks like some definition had its argument order changed, could you note this in the commit message?

Reply:
Done.